usr/lib/linuxmint/mintbackup/mintbackup.py

```diff
@@ -62,7 +62,7 @@ def __init__(self):
         self.builder.set_translation_domain(APP)
         self.builder.add_from_file(UI_FILE)
 
-        self.settings = Gio.Settings("com.linuxmint.backup")
+        self.settings = Gio.Settings(schema="com.linuxmint.backup")
```

Comment:
Shouldn't this be schema_id?  Weird they'd complain about using positional argument but don't care if you use a deprecated property to replace it. :)